experiments/browser_based_querying/package.json

```diff
@@ -30,18 +30,22 @@
   "dependencies": {
     "@emotion/react": "^11.9.0",
     "@emotion/styled": "^11.8.1",
+    "@graphiql/react": "^0.19.3",
     "@mui/icons-material": "^5.10.2",
     "@mui/lab": "^5.0.0-alpha.92",
     "@mui/material": "^5.8.0",
     "core-js": "^3.22.5",
     "detect-browser": "^5.3.0",
-    "graphiql": "^1.11.5",
```

Comment:
Yes, it's not used. I think it was conflicting with something so I removed it, but it doesn't matter really since it's not used. I can add it back then remove it in another pr if that would be better.